packages/grafbase-sdk/tests/unit/federation.test.ts

```diff
@@ -0,0 +1,42 @@
+import { config, g } from '../../src/index'
+import { describe, expect, it, beforeEach } from '@jest/globals'
+import { renderGraphQL } from '../utils'
+
+describe('Federation generator', () => {
+  beforeEach(() => g.clear())
+
+  it('renders federation when enabled', async () => {
+    g.type('Post', {
+      id: g.id()
+    })
+    const cfg = config({
+      schema: g,
+      federation: {
+        version: '2.3'
+      }
+    })
+
+    expect(renderGraphQL(cfg)).toMatchInlineSnapshot(`
+      "type Post {
+        id: ID!
+      }
+      extend schema @federation(version: "2.3")
+      "
+    `)
+  })
+
+  it('does not render federation when disabled', async () => {
```

Comment:
Is this necessary? We have plenty of tests where we don't have federation set and they should show breakage.

Reply:
Fair enough

Reply:
Removed